examples/CMakeLists.txt

```
@@ -447,12 +447,30 @@ add_impactx_test(solenoid.MADX.py
     OFF  # no plot script yet
 )
 
+# Pole-face rotation ###################################################
+#
+# w/o space charge
+add_impactx_test(rotation
+    examples/rotation/input_rotation.in
+      ON  # ImpactX MPI-parallel
+      OFF  # ImpactX Python interface
+    examples/rotation/analysis_rotation.py
+    OFF  # no plot script yet
+)
+add_impactx_test(rotation.py
+    examples/rotation/run_rotation.py
+      OFF  # ImpactX MPI-parallel
+      ON   # ImpactX Python interface
+    examples/rotation/analysis_rotation.py
+    OFF  # no plot script yet
+)
+
 # Soft-Edge Solenoid ###################################################
 #
 # w/o space charge
 add_impactx_test(solenoid_softedge
     examples/solenoid_softedge/input_solenoid_softedge.in
-      OFF  # ImpactX MPI-parallel
+      ON  # ImpactX MPI-parallel
```

Since this is another test, was this unintentionally changed in this PR?
```suggestion
      OFF  # ImpactX MPI-parallel
```